src/components/custom/RejectedBadge.js

```diff
@@ -0,0 +1,7 @@
+export function RejectedBadge() {
+	return (
+		<div className="justify-center items-center h-fit px-2 py-0.5 rounded-lg text-sm bg-red-300 text-red-600">
+			Published
```

Comment:
This shouldn't be `Published` but should be `Rejected`. Also, the styling of this is almost the same as the previous - we shouldn't be making a new component for this. We also need badges for `Contributed` and `Not contributed` looking at the figma.

Reply:
Resolved